packages/react-lowcode/src/codegen/index.ts

```diff
@@ -47,8 +47,26 @@ export function generatePages(inputSourceCode: string, io: CodeRW & CodeDir, opt
                 true,
                 ts.ScriptKind.TSX
             )
-            const pageSouceCode = printer.printList(ts.ListFormat.MultiLine, factory.createNodeArray([...page.imports, page.functionDeclaration]), sourceFile)
+            const pageSouceCode = printer.printList(ts.ListFormat.MultiLine, factory.createNodeArray([...page!.imports, page!.functionDeclaration]), sourceFile)
             io.writeFile(filePath, pageSouceCode)
+
+            //generate list wrapper
+            const indexWrapperTemplatePath = 'path-to-template'//TODO: put here real template path when template will be done
+            let template = ''
+            io.readFile(indexWrapperTemplatePath).then((source => {if(source) template = source;}))
+
+            const listWrapper = generator.generateListComponentWrapper(template)
+            const listWrapperFilePath = `src/components/${typeName}Wrapper.tsx`//TODO: dont like the word wrapper, rename later to something else
```

Comment:
Let's have naming strategy (interface and implementation) for:
* component name
* file name

Please, check the discord channel for the file naming conventions from different projects.